scopes/cloud/cloud/cloud.main.runtime.ts

```diff
@@ -0,0 +1,85 @@
+import { MainRuntime } from '@teambit/cli';
+import {
+  BASE_CLOUD_DOMAIN,
+  DEFAULT_HUB_DOMAIN,
+  SYMPHONY_URL,
+  SYMPHONY_GRAPHQL,
+  DEFAULT_HUB_LOGIN,
+  DEFAULT_ANALYTICS_DOMAIN,
+  DEFAULT_REGISTRY_URL,
+  CENTRAL_BIT_HUB_URL,
+  CENTRAL_BIT_HUB_NAME,
+} from '@teambit/legacy/dist/constants';
+
+import { CloudAspect } from './cloud.aspect';
+
+export interface CloudWorkspaceConfig {
+  cloudDomain: string;
+  cloudHubDomain: string;
+  cloudApi: string;
+  cloudGraphQL: string;
+  loginDomain: string;
+  analyticsDomain: string;
+  registryUrl: string;
+  cloudExporterUrl: string;
+  cloudHubName: string;
+}
+
+export class CloudMain {
+  constructor(private config: CloudWorkspaceConfig) {}
+
+  getCloudDomain(): string {
+    return this.config.cloudDomain;
+  }
+
+  getCloudHubDomain(): string {
```

Comment:
Not sure we need all getters. Maybe just do `readonly config`

Reply:
I don't want you to access it as cloud.config.hubDomain. don't want you to use the config in the middle. it should be hidden from you